wine-tkg-git/PKGBUILD

```diff
@@ -148,12 +148,22 @@ else
   fi
 
   if [ "$_use_dxvk" == "true" ] && [ "$_use_dxup" == "true" ]; then
-    pkgname="${pkgname/%-git/-dxvk+up-git}"
+		if [ "$_dxvk_async" == "true"]; then
```

Comment:
That whole block seems unnecessary to me. Adding a tag instead would be enough.

Reply:
I see now that you did add the tag. I'm convinced you should still consider dropping this block.

Reply:
Dropped.